run_nodes.sh

```diff
@@ -35,6 +35,7 @@ for i in ${!authorities[@]}; do
     --chain dev \
     --base-path /tmp/$auth \
     --name $auth \
+    --rpc-port $(expr 9933 + $i) \
```

Comment:
is it possible to move these port ranges 9933 --- 9944 further apart? I would imagine it's possible that someday one would run it with more than 10 nodes and then there could be some collisions (not sure how dangerous this is though :) )

Reply:
Port 9933 is the default port for the HTTP RPC endpoint, so I would prefer to leave it as is. Especially since this is only intended to run nodes on a single machine, so I wouldn't expect anyone running more than 10.

In good news I'm pretty sure collisions don't break much  – without this argument all nodes tried starting their HTTP RPC endpoint on port 9933 and one of them always succeeded, the rest failed silently. So even if someone runs more than 10 nodes at some point, they only have to make sure that they attempt HTTP RPC connections to one of the first ports and the WebSocket RPC connections to one of the last.